### DISCUSSION TITLE
Connection with VLC success but with synology fails

### DISCUSSION THREAD

**Discussion by purple-box-uk** [bluenviron/mediamtx#3129] [Questions]
### Question

I can successfully see the stream on VLC on another machine after running mediaMTX and ffmpeg command, but when i paste the same URL into synology then i see 'CLOSED invalid URL (/mystream)' in the mediaMTX console. I can't figure out why, anyone with any idea of what to check?

the FFMPEG command: ffmpeg -f gdigrab -framerate 30 -i desktop -vcodec libx264 -preset ultrafast -tune zerolatency -f rtsp rtsp://172.18.61.5:8554/mystream

rtsp://172.18.61.5:8554/mystream works in VLC

**Comment by aler9** [comment]
Open a bug report and provide a full server log with `logLevel: debug`, as explicitly mentioned in the instructions. Otherwise this problem cannot be replicated nor solved.
